domain/schema/model/sql/0018-machine.sql

```diff
@@ -27,8 +26,29 @@ ON machine (name);
 CREATE UNIQUE INDEX idx_machine_net_node
 ON machine (net_node_uuid);
 
--- machine_parent table is a table which represents parents-children relationships of machines.
--- Each machine can have a single parent or be a parent to multiple children.
+CREATE TABLE machine_placement_scope (
+    id INT PRIMARY KEY,
+    value TEXT NOT NULL
+);
+
+INSERT INTO machine_placement_scope VALUES
```

Comment:
Worth a note that the other scope is _machine_, but that identifies a specific machine and so does not end up on the machine itself. Rather than omitting it, we're keeping this in the even that scopes are expanded.